coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/VersionedSqlUserDefinedFunction.java

```diff
@@ -68,6 +87,27 @@ public String getViewDependentFunctionName() {
     return viewDependentFunctionName;
   }
 
+  /**
+   * Retrieves the short function name based on the class name. If the class name is found
+   * in the predefined {@link VersionedSqlUserDefinedFunction#SHORT_FUNC_NAME_MAP},
+   * the corresponding short name is returned. Otherwise, the method converts the last
+   * segment of the class name from UPPER_CAMEL to LOWER_UNDERSCORE format to generate
+   * the short function name.
+   */
+  public String getShortFunctionName() {
+    String unversionedClassName = getName();
```

Comment:
Please add java doc about the nature of the function name (e.g., its format etc). You can say this is just a convention and other naming approaches are still valid as long as they identify the type inference.

Reply:
Done